libs/libxx/libcxx.defs

```diff
@@ -0,0 +1,50 @@
+############################################################################
+# libs/libxx/libcxx.defs
+#
+# Licensed to the Apache Software Foundation (ASF) under one or more
+# contributor license agreements.  See the NOTICE file distributed with
+# this work for additional information regarding copyright ownership.  The
+# ASF licenses this file to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance with the
+# License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+# WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.  See the
+# License for the specific language governing permissions and limitations
+# under the License.
+#
+###########################################################################
+
+VERSION=11.0.0
+
+$(TOPDIR)/include/libcxx:
+	$(Q) wget https://github.com/llvm/llvm-project/releases/download/llvmorg-$(VERSION)-rc1/libcxx-$(VERSION)rc1.src.tar.xz
```

Comment:
is it ok that this is based around the rc1 of every version? maybe rc1 should be part of VERSION

Reply:
I have to move rc1 out of VERSION, because the directory contain '-'(llvmorg-$(VERSION)-rc1), but the file name don't(libcxx-$(VERSION)rc1.src.tar.xz).

Reply:
Maybe something like defining `EXTRAVERSION=rc1` and build the required tar and directory names from that.

Reply:
rc1 is a special case, I have to use rc1 becasue it's more match our code base than 10.0.1. In the furture, it's better to align with the official release instead of release candidate. So rc1 will disappear once libc++ 11.0.0 is out.

Reply:
Ok, I agree